bundle.js

```diff
@@ -1,2 +1,2 @@
d))return null;for(const o of null==e?void 0:e.configFiles)if(t.endsWith(o))return x(n,e,t);return null}async function O(e,n){if(!e.outputLocation)return'./';if(!1===e.outputLocation.includes('{'))return e.outputLocation;const t=e.outputLocation.indexOf('{'),o=e.outputLocation.indexOf('}'),i=e.outputLocation.substring(t+1,o).trim(),[a,r]=i.split('#'),c=v.join(n,a.trim()),u=await h(c);if(u){if(j(u)){const e=k(u),n=await async function(e,n){return s(e,n)}(e,r.trim())||'./';return n}console.warn(`[${e.name}] Could not parse JSON file: ${c}`)}return'./'}function S(e,n){const t=n.find((n=>n.framework.id===e.framework.id));return t?t.matchedFiles=[...new Set([...t.matchedFiles,...e.matchedFiles])]:n.push(e),n}async function x(e,n,t){const o=l(t),i=function(e){if((e=l(e)).startsWith('http')){const n=f();if(n){const{branch:t,url:o}=n.$$repo;return v.dirname(e.replace(o,'').replace(`/blob/${t}/`,'/'))}return e}return v.dirname(e)}(t),{repo:a,owner:r}=function(e){const[n,t,o,i,a]=e.split('/');return{repo:a,owner:i}}(e);return{repo:a,owner:r,deployment:{appLocation:i,outputLocation:await O(n,e)},matchedFiles:[o],framework:n}}(e=>{e.static='static',e.nodejs='nodejs',e.dotnet='dotnet',e.python='python',e.django='django',e.angular='angular',e.angularUniversal='angular-universal',e.ionicAngular='ionic-angular',e.scully='scully',e.react='react',e.preact='preact',e.vue='vue',e.nuxtjs='nuxtjs',e.vuepress='vuepress',e.aurelia='aurelia',e.elm='elm',e.ember='ember',e.flutter='flutter',e.hugo='hugo',e.knockoutjs='knockoutjs',e.lit='lit',e.marko='marko',e.meteor='meteor',e.mithril='mithril',e.polymer='polymer',e.riot='riot',e.stencil='stencil',e.svelte='svelte',e.typescript='typescript',e.azureFunctions='azure-functions',e.dockerFile='docker-file'})(b||(b={}));const T=(e,n,t)=>`https://github.com/${e}/${n}${t?`#${t}`:''}`,N=require('express'),E=require('cors'),I=N(),J=process.env.PORT||8080;I.use(E({origin:['*']})),I.get('/',(async(e,n)=>{const t=e.query.repo,o=e.query.org;let i=e.query.branch;const a=e.headers['github-token'];if(process.env.GITHUB_TOKEN='',process.env.__GITHUB_PROJECT__='',o&&'string'==typeof o)if(t&&'string'==typeof t){a&&'string'==typeof a?process.env.GITHUB_TOKEN=a:console.warn('Invalid token header'),'string'!=typeof i&&(i='',console.warn('Invalid branch parameter'));try{await P(n,o,t,i)}catch(e){n.status(500),n.json(JSON.parse(JSON.stringify(e)))}}else n.status(400),n.json({message:'Invalid or missing parameters: Repo'});else n.status(400),n.json({message:'Invalid or missing parameters: Org'})})),I.listen(J,(()=>{console.log(`⚡️[server]: Server is running at https://localhost:${J}`)}));const P=(e,n,t,o)=>((e,n,t)=>L(T(e,n,t),[],!0,[]))(n,t,o).then((i=>{((e,n,t,o)=>{const i=T(n,t,o);return Promise.resolve(w(e,i))})(i,n,t,o).then((n=>{e.status(200),e.json({frameworks:i,recommendation:n})})).catch((n=>{console.log('Error on Recommendation: '+n),e.status(500),e.json(n)})).catch((n=>{console.log('Error on framework: '+n),e.status(500),e.json(n)}))}));
```

Comment:
remove this change?

Reply:
Those are all somehow autogenerated. I will remove them

Reply:
Done